src/views/LaunchableMarketStatsDetails.tsx

```diff
@@ -54,17 +55,32 @@ export const UnlaunchedMarketStatsDetails = ({
   const {
     marketCap,
     price,
+    reportedMarketCap,
     tickSizeDecimals,
     volume24h: spotVolume24H,
   } = orEmptyObj(launchableAsset);
 
+  const showSelfReportedMarketCap = marketCap ? false : !!reportedMarketCap;
+
   const valueMap = {
-    [MarketStats.MARKET_CAP]: marketCap,
+    [MarketStats.MARKET_CAP]: showSelfReportedMarketCap ? reportedMarketCap : marketCap,
     [MarketStats.SPOT_VOLUME_24H]: spotVolume24H,
   };
 
+  const tooltipMap = {
+    [MarketStats.MARKET_CAP]: showSelfReportedMarketCap
+      ? ('self-reported-cmc' as TooltipStringKeys)
+      : undefined,
+    [MarketStats.SPOT_VOLUME_24H]: undefined,
+  };
```

Comment:
maybe use a satisfies here and we don't need the cast as unknown below.

`satisfies Record<MarketStats, TooltipStringKeys>` maybe?